### PR TITLE
SEG-699: Provide public documentation for new GraphQL endpoints

### PR DESCRIPTION
**Pull request by shiftedreality**
## Purpose of this pull request

This pull request (PR) is an documentation for https://jira.corp.adobe.com/browse/SEG-699

## Affected pages

- http://localhost:8000/graphql/schema/store/queries/dynamic-blocks/

### CODE REVIEW

**Review by erikmarr — APPROVED** [review]

File: src/pages/graphql/schema/store/queries/dynamic-blocks.md [hunk 1]
```diff
@@ -151,6 +152,7 @@ Attribute | Data type | Description
 --- | --- | ---
 `content` | ComplexTextValue! | Contains the renderable HTML code of the dynamic block
 `uid` | ID! | The unique ID of a DynamicBlock object
+`audeince_id` | [ID] | An array of of Audience IDs for this block. Available in the `magento/audiences` module only.
```
Comment (erikmarr) [inline src/pages/graphql/schema/store/queries/dynamic-blocks.md:155]:
```suggestion
`audience_id` | [ID] | An array of of Audience IDs for this block. Available in the `magento/audiences` module only.
```

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
Holding off on merging until product releases.

**Comment by keharper** [comment]
@shiftedreality Please alphabetize all new fields.

**Comment by keharper** [comment]
@shiftedreality  If this is going out at the same time as the 2.4.7 beta release, please rebase this PR to `2.4.7-develop`. Then @erikmarr can go ahead and merge the PR (after the fields are alphabetized).

**Comment by shiftedreality** [comment]
@keharper This will be released on May 30

**Pull request merged**